docker-compose.yml

```diff
@@ -3,17 +3,38 @@ version: '3'
 services:
     mongodb:
         image: mongo
+        command: [ "/usr/bin/mongod", "--replSet", "projectsRepSet" ]
+        ports:
+            - "27017:27017"
+        restart: "on-failure"
+
+    mongodb_replica_setup:
+        build: ./backend/mongodb_replica_setup
+        depends_on:
+            - "mongodb"
 
     elasticsearch:
-        image: elasticsearch
+        image: docker.elastic.co/elasticsearch/elasticsearch:5.4.1
+        environment:
+            - "http.host=0.0.0.0"
+            - "transport.host=127.0.0.1"
+            - "xpack.security.enabled=false"
+            - "action.auto_create_index=true"
         ports:
             - "9200:9200"
+        restart: "on-failure"
```

Comment:
we should test whether an elasticsearch error inducing a shutdown returns a failure code, mongo-connector for example doesn't and has to be on "always"